Failure when running unit tests after clone of master

```
======================================================================
FAIL: test_pushGetSwapped (test_orderedBuffer.TestOrderedBuffer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jamesba/src/rd-apmm-python-lib-rtpTTML/tests/test_orderedBuffer.py", line 125, in test_pushGetSwapped
    st.integers(min_value=0, max_value=MAX_SEQ_NUM),
  File "/home/jamesba/src/rd-apmm-python-lib-rtpTTML/.tox/unittest/lib/python3.6/site-packages/hypothesis/core.py", line 1032, in wrapped_test
    raise the_error_hypothesis_found
  File "/home/jamesba/src/rd-apmm-python-lib-rtpTTML/tests/test_orderedBuffer.py", line 147, in test_pushGetSwapped
    self.assertEqual(list(expectedList.values()), receivedList)
AssertionError: Lists differ: [<rtp[65 chars]cc75ecc0>, <rtp.rtp.RTP object at 0x7f8ccc75e4[279 chars]828>] != [<rtp[65 chars]cc75e438>, <rtp.rtp.RTP object at 0x7f8ccc75ec[279 chars]828>]

First differing element 1:
<rtp.rtp.RTP object at 0x7f8ccc75ecc0>
<rtp.rtp.RTP object at 0x7f8ccc75e438>

  [<rtp.rtp.RTP object at 0x7f8ccc75ee80>,
+  <rtp.rtp.RTP object at 0x7f8ccc75e438>,
   <rtp.rtp.RTP object at 0x7f8ccc75ecc0>,
-  <rtp.rtp.RTP object at 0x7f8ccc75e438>,
   <rtp.rtp.RTP object at 0x7f8ccc7f5e80>,
   <rtp.rtp.RTP object at 0x7f8ccc7f50f0>,
   <rtp.rtp.RTP object at 0x7f8ccc7f5e10>,
   <rtp.rtp.RTP object at 0x7f8ccc7c4320>,
   <rtp.rtp.RTP object at 0x7f8ccc7c4eb8>,
   <rtp.rtp.RTP object at 0x7f8ccc7c4860>,
   <rtp.rtp.RTP object at 0x7f8ccc7c4828>]

----------------------------------------------------------------------
Ran 15 tests in 2.720s

FAILED (failures=1)
```

--- a/tests/test_orderedBuffer.py
+++ b/tests/test_orderedBuffer.py
@@ -127,21 +127,23 @@ class TestOrderedBuffer (TestCase):
                  min_size=2, max_size=2, unique=True))
     def test_pushGetSwapped(self, startKey, swap):
         self.assertFalse(self.buffer.available())
-        expectedList = {}
+        expectedList = []
         receivedList = []
 
-        keyOffsets = list(range(10))
-        keyOffsets[swap[0]], keyOffsets[swap[1]] = (
-            keyOffsets[swap[1]], keyOffsets[swap[0]])
-
-        for x in keyOffsets:
+        for x in list(range(10)):
             seqNum = (startKey + x) % (MAX_SEQ_NUM + 1)
             packet = RTP(sequenceNumber=seqNum)
-            expectedList[x] = packet
-            ret = self.buffer.pushGet(seqNum, packet)
+            expectedList.append(packet)
+
+        swappedPackets = expectedList.copy()
+        swappedPackets[swap[0]], swappedPackets[swap[1]] = (
+            swappedPackets[swap[1]], swappedPackets[swap[0]])
+
+        for packet in swappedPackets:
+            ret = self.buffer.pushGet(packet.sequenceNumber, packet)
             receivedList += ret
 
         self.assertFalse(self.buffer.available())
 
         self.assertEqual(len(expectedList), len(receivedList))
-        self.assertEqual(list(expectedList.values()), receivedList)
+        self.assertEqual(expectedList, receivedList)

